scripts/netContain/build_image.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+BUILD_ENV_IMAGE_NAME="contiv-netplugin-build"
+TEMPORARY_CONTAINER_NAME="contiv-netplugin-build-temporary"
+FINAL_IMAGE_NAME="contivbase"
+
+NETPLUGIN_BINARIES="netplugin netmaster netctl contivk8s"
+
+function get_image_id() {
+	docker inspect --format '{{.ID}}' $1
+	return $?
+}
+
+function build_image_from_path() {
+	set +e
+	old_image=$(get_image_id $2)
+	set -e
+	if [ "$?" -ne 0 ]; then
+		old_image=
+	fi
+
+	echo "Building image $2 from $1 ..."
+	docker build -t $2 --force-rm $1
+	if [ "$?" -ne 0 ]; then
+		echo "Image $2 was not built properly"
+		return 1
+	fi
+
+	new_image=$(get_image_id $2)
+	if [ "$old_image" != "" ] && [ "$old_image" != "$new_image" ]; then
+		docker rmi $old_image >/dev/null 2>&1 || true
+	fi
+
+	echo "Image $2 has been built"
+	return 0
+}
+
+set -x
+set -e
+
+rm -rf scripts/netContain/bin
```

Comment:
Clean up the temporary bin directory. This directory is used to make it easier to remove the binaries and to ensure that we don't leave binaries around for no good reason.